scripts/jenkins-test

```diff
@@ -2,81 +2,107 @@
 
 set -e -x
 
+export JAVA_HOME=/usr/java/jdk1.8.0_60
```

Comment:
put all of your variable decs up here, rather than spreading them throughout the script.  and only use export for variables that need to be visible to subprocs (JAVA_HOME, PATH, etc).